### PR TITLE
I created and tweaked the components contactForm

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

**Review by KeerhemFR — DISMISSED** [review]
I see no evil here

File: src/components/Contact/ContactForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import SFormContact from './style';
+
+export default function ContactForm() {
+  const [form, setForm] = useState({
+    name: '',
```
Comment (Hacene010) [inline src/components/Contact/ContactForm/index.jsx]: lastname is more appropriate

File: src/components/Contact/ContactForm/index.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        Nom :
+        <input type="text" name="name" onChange={handleChange} />
+      </label>
+      <label htmlFor="firstname">
+        Prénom :
+        <input type="text" name="firstname" onChange={handleChange} />
+      </label>
+      <label htmlFor="email">
+        Email :
+        <input type="text" name="email" onChange={handleChange} />
```
Comment (Hacene010) [inline src/components/Contact/ContactForm/index.jsx]: Put the type ="email" here. It's better

File: src/components/Contact/ContactForm/index.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </label>
+      <label htmlFor="message">
+        Message :
+        <textarea id="msg" name="message" onChange={handleChange} />
```
Comment (Hacene010) [inline src/components/Contact/ContactForm/index.jsx]: What is this id for ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Seen some evils. See the comments

**Review by KeerhemFR — APPROVED** [review]
The requested changes were corrected

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**